components/automate-gateway/api/iam/v2/teams.proto

```diff
@@ -4,14 +4,75 @@ package chef.automate.api.iam.v2;
 option go_package = "github.com/chef/automate/components/automate-gateway/api/iam/v2";
 
 import "google/api/annotations.proto";
+import "protoc-gen-swagger/options/annotations.proto";
 
 import "components/automate-gateway/api/iam/v2/request/teams.proto";
 import "components/automate-gateway/api/iam/v2/response/teams.proto";
 import "components/automate-grpc/protoc-gen-policy/api/annotations.proto";
 import "components/automate-grpc/protoc-gen-policy/iam/annotations.proto";
 
 service Teams {
+  /*
+  Creates a local team
+
+  Creates a local team that is used to group local users as members of IAM policies.
+
+  Authorization Action:
+  ```
+  iam:teams:create
+  ```
+  */
+  rpc CreateTeam (CreateTeamReq) returns (CreateTeamResp) {
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
```

Comment:
I see this appearing twice, once right here and once for `CreateTeamReq` in components/automate-gateway/api/iam/v2/request/teams.proto. I saw something in the backscroll about it but still not clear. Is one for the "Payload" and the other for the "JSON" choices in "Request samples" ?